library/oneview_scope.py

```diff
@@ -145,6 +145,11 @@ def execute_module(self):
             return self.__update_resource_assignments()
 
     def __update_resource_assignments(self):
+        # returns None if scope doesn't exist
+        if not self.current_resource:
+            return dict(changed=False,
+                        msg=self.MSG_RESOURCE_NOT_FOUND)
```

Comment:
@VenkateshRavula I tried running this example, but this results to an ok state, if ran without verbose mode, doesn't provides me the error for scope not found, i believe it will leave user ambiguous. 
I would suggest to return failed=True, with the requested change in place, User will get an error statement.

Reply:
Done